src/app/services/xml-parsers/witnesses-parser.service.ts

```diff
@@ -0,0 +1,134 @@
+import { Injectable } from '@angular/core';
+import { map, shareReplay } from 'rxjs/operators';
+import { Description, Witness, WitnessesData, WitnessGroup, WitnessGroups, XMLElement } from '../../models/evt-models';
+import { isNestedInElem, xpath } from '../../utils/dom-utils';
+import { replaceNotWordChar } from '../../utils/xml-utils';
+import { EditionDataService } from '../edition-data.service';
+import { GenericParserService } from './generic-parser.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class WitnessesParserService {
+  public readonly parsedList$ = this.editionDataService.parsedEditionSource$.pipe(
+    map((source) => this.parseWitnessesData(source)),
+    shareReplay(1),
+  );
+
+  private witListTagName = 'listWit';
+  private witTagName = 'witness';
+  private witNameAttr = 'type=\'siglum\'';
```

Comment:
Use backticks in order to avoid the back slashes: ``` `type='siglum'` ```, or just use double quotation marks inside the single ones `'type="siglum"'`.